src/main/java/core/dao/FileReaderService.java

```diff
@@ -0,0 +1,5 @@
+package core.dao;
+
+public interface FileReaderService {
```

Comment:
let's refactor this service - make the method read() only return List of strings that were read from the file